pkg/rest/config_upload.go

```diff
@@ -57,6 +59,18 @@ func upsertDynatraceObject(client *http.Client, fullUrl string, objectName strin
 		if err != nil {
 			return api.DynatraceEntity{}, err
 		}
+
+		if success(resp) {
+			util.Log.Debug("\t\t\tUpdated existing object for %s (%s)", objectName, existingObjectId)
+			return api.DynatraceEntity{
+				Id:          existingObjectId,
+				Name:        objectName,
+				Description: "Updated existing object",
+			}, nil
+		} else {
+			return dtEntity, fmt.Errorf("Failed to update DT object %s (HTTP %d)!\n    Response was: %s", objectName, resp.StatusCode, string(resp.Body))
```

Comment:
Just to be consistent, could you please return `api.DynatraceEntity{}` here? 
(Also to make it more explicit)

Reply:
it will already be that from the var on line 33

But yes, we seem to use that very inconsistently in this method (and we're generally inconsistent e.g. on named return values)

Maybe a codestyle thing we should tackle separate from this?

Reply:
Yes, but if anybody would set a property on this var, we would return it without noticing ^^

I agree, that it is a code style topic, but to be at least consistent within the method, 
we should return a new empty struct.

Reply:
done